### ISSUE TITLE
Simplify Authentication Workflow

### ISSUE THREAD

**Issue by benrules2** [dend/foggycam#72]
Populating the configuration is challenging, especially for non-tech savvy users. Improvements to the authentication workflow would be much appreciated. 

Some possible improvements of the top of my head:
- Auto-detect the `iframerpc` calls of interest for the cookie and `issue_token` 
- Initiate a web session automatically, and scrape everything from the cookies automatically (best case)
- Automate the auth workflow with selenium, and download the interesting network activity to disk?

*This is related to the UBC Animal Welfare collaboration effort

**Comment by dend** [comment]
Yes, this change makes sense. I am looking at potentially simplifying how the tokens are acquired through the Google authorization workflows. There are promising leads, but I need to spend some more time investigating.
